content/docs/querying/functions.md

```diff
@@ -212,8 +216,9 @@ increase(http_requests_total{job="api-server"}[5m])
 ```
 
 `increase` should only be used with counters. It should be used primarily for
-human readability. Use `rate` in recording rules so that increases are tracked
-consistently on a per-second basis.
+human readability and is really just a syntactic sugar around `rate` times the
```

Comment:
Avoid words like "really" in reference docs. Better:

"It is syntactic sugar for `rate(v)` multiplied by the number of seconds under the specified range vector time window. It should be used primarily for human readability."